src/components/Clock/styles.ts

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+
+export const ClockContainer = styled.div`
+  background-color: #cb6e6e;
+  display: grid;
+  justify-content: center;
+  border-radius: 5px;
+  width: 30rem;
+  padding: 0rem 1rem 2rem 1rem;
```

Comment:
[suggestion] You can add a top padding here instead of adding a top margin to the reset button. Usually we don't define margins to components as they can be used in different places and need different margins, this way it's better to set a padding to the container rather than margin to the element itself.